### PR TITLE
Peer review

### PR DESCRIPTION
**Pull request by souad988**
Updated to be responsive to desktop version

### CODE REVIEW

**Comment by Ninjaphil24** [comment]
The code generally looks good other than the linter error.  I'm not sure I would use two different css files for desktop and mobile.  The main reason being that further down the line you may need more media queries than just those two, so you'll end up using media queries inline anyway.

**Comment by i-max-xi** [comment]
Hi @souad988 great job so far 👍.
Just a few checks to do

Required Changes ♻️ 
Please make sure all linter checks have passed.
Please refer to the attached image below for clarity.
linter

So you could click on "details" on the right section to access the required stylelint changes.
Everything else looks good 👍 . Happy coding

File: src/styles.css [hunk 1]
```diff
@@ -33,11 +37,13 @@
 
 .home {
   background-image: url("../media/images/header-illsutration-mobile@2x.png");
-  background-size: cover;
+  background-size: contain;
+  background-repeat: no-repeat;
+  background-position-x: right ;
```
Comment (Ninjaphil24) [inline src/styles.css:42]: An easier way to do back ground is shorthand, which would be "background-position: 100% 100%;" (or a different %)
That way you can move things around easier, and also test it easier with devtools in the browser.

**Review by Ninjaphil24 — COMMENTED** [review]
Over all very good.  Left a couple of optional changes.

File: src/styles.css [hunk 2]
```diff
@@ -138,7 +154,7 @@
 
 .card {
   flex-direction: column-reverse;
-  min-height: 75vh;
+  min-height: 60vh;
 }
 
 .card_info {
```
Comment (mwafrika) [inline src/styles.css:160]: Please, remove empty CSS rules in the project to not confuse yourself,

Comment (souad988) [reply src/styles.css:160]: thanks for reviewing, can you specify please wich rule is empty?

File: src/styles.css (hunk 1 above)
Comment (souad988) [reply src/styles.css]: thanks it worked great

**Pull request merged**